gcp_variant_transforms/transforms/merge_headers.py

```diff
@@ -13,26 +13,174 @@
 # limitations under the License.
 
 """beam combiner function for merging VCF file headers."""
-
+from collections import OrderedDict
 import apache_beam as beam
-from gcp_variant_transforms.beam_io import vcf_header_io
 
-__all__ = ['MergeHeaders']
+from gcp_variant_transforms.beam_io.vcf_header_io import VcfHeader
+
+__all__ = ['HeaderConflictResolver', 'HeaderMerger', 'MergeHeaders']
+
+class HeaderConflictResolver(object):
+  """A class for resolving mistmatch between header fields (e.g. type, num).
```

Comment:
nit: the closing """ can fit on the same line?

Reply:
Done